src/components/Person/_person.scss

```diff
@@ -105,17 +111,35 @@
   }
 }
 
-.cc-person--author .cc-person__image + .cc-person__wrapper {
-  @include mq(sm) {
-    flex-basis: calc(100% - var(--person-image-size-small));
-  }
-}
-
 .cc-person--author .cc-person__heading {
   font-size: var(--heading-xs);
 }
 
 .cc-person--author .cc-person__description,
-.cc-person--author a {
+.cc-person--author .cc-person__link {
   font-size: var(--body-sm);
 }
+
+// variant: NARROW
+
+.cc-person--narrow .cc-person__image {
+  @include mq(md) {
+    position: absolute;
+  }
+
+  @include mq(xl) {
+    position: static;
+  }
+}
+
+.cc-person__image--narrow + .cc-person__wrapper .cc-person__header {
```

Comment:
CSS selectors this specific make me nervous 😬 You never know what issues they might cause further down the line (e.g. if trying to apply a utility selector to the same element, which selector takes specificity?).

I can appreciate why you've done this, as opposed to applying a variant selector to `cc-person__header`, so I'll let you decide what you think is best - just wanted to flag my concerns with very specific CSS selectors 😅